bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java

```diff
@@ -178,6 +181,35 @@ protected void onLayout(boolean changed, int left, int top, int right, int botto
         this.contentClipRect.set(0, 0, getWidth(), bottomClip);
     }
 
+    @Override
+    public boolean onKeyPreIme(int keyCode, KeyEvent event) {
```

Comment:
Code taken from stock widgets, so I figured why change something tested. This isn't working properly anyway, so I closed the pull request.

Btw - why don't you submit a pull request if you know how to do this?